### PR TITLE
Updated README.md - Add additional introductory paragraph

### PR DESCRIPTION
**Pull request by frankstepanski**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?**

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

**Comment by frankstepanski** [comment]
Updated introductory paragraph to README for further clarification of how Angular works.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ synchronizes data from your UI (view) with your JavaScript objects (model) throu
 binding. To help you structure your application better and make it easy to test, AngularJS teaches
 the browser how to do dependency injection and inversion of control.
 
+HTML is also used to determine the execution of the app. Special attributes in the HTML determine which controllers to use for each element. These attributes determine "what" gets loaded, but not "how". This declarative approach greatly simplifies app development in a sort of WYSIWYG way. Rather than spending time on how the program flows and what should get loaded first, you simply define what you want and Angular will take care of the dependencies.
```
Comment (gkalpak) [inline README.md]: I feel this goes into a bit too much detail for this high-level description here.
Maybe incorporate it into the `Interconnection with HTML at the root level` section below (or in it's own section).

**Comment by Narretz** [comment]
@frankstepanski Can you incorporate the changes @gkalpak requested? Otherwise we'll have to close this PR.

**Comment by frankstepanski** [comment]
Sorry, missed this...will incorporate it his weekend. thanks.

**Comment by frankstepanski** [comment]
Updated

**Comment by gkalpak** [comment]
I tweaked it a bit and merged. Thx!
<sub>(I also seized the opportunity to limit all lines to 100 chars :grin:)</sub>

**Comment by frankstepanski** [comment]
Thanks!

**Pull request closed without merging**